README.md

```diff
@@ -1 +1,177 @@
+
+### Deepstack-CameraUI Settings
+The side-bar compresses different elements detailed below:
+
+* **Choose the APIs**
+
+This widject allows you to select which API techniques delivered by this application to run among the following **object detection**, **face recognition**, **face registration**, **face deregistration**, **face detection** and **custom detection**.
+
+* **DeepStack URL**
+
+To connect to Deepstack server the user must provide the right URL to Deepstack like `http://localhost:80` or `http://localhost:5670` depending on the URL port. 
+Fro more information check the documentation https://docs.deepstack.cc/
+
+* **Camera Address**
+
+This is the crucial part of this application, the user must supply all the information related to his IP canera such **User name**, **password**, **IP address**, **port** and **media**.
+
+- User name: A string to identify oneself i.e: nikeo, paul 
+- Password can be anything.
+- IP address *: the Internet Protocol of the camera should be in the format `192.168.1.199`
+- Port* : represents the LAN port i.e: 8900, 8081
+- Video stream path: used to access the ip camera video can be `video` or `media` etc.
+
+The terms ending with * are **compulsory**.
+
+When the user has a password and name, the camera address should have the following format
+```
+admin:password@IP Address:Port/Video stream path
+```
+Otherwise,
+```
+IP Address:Port/Video stream path
+```
+
+
+* **Custom Model Name**
+
+This widget accepts the name of the custom model, its is optional only required when custom detection is selected.
+
+* **Webhook URL**
+
+The URL inputted will receive the detailed informations about the content of the detections being captured in real-time and the api running.
+
+* **Webhook Api Key**
+
+The definition here
+
+* **Webhook Forward Images**
+
+Here the user can decide to receive the images of the live detection through the webhook URL or not.
+
+
+## Installations 
+
+In order to install **Python 3.8** depending on any operating system 
+
+- [Windows](https://www.python.org/ftp/python/3.8.0/python-3.8.0-amd64.exe) 
+- [MacOS](https://www.python.org/ftp/python/3.8.0/python-3.8.0-macosx10.9.pkg)
+- [Ubuntu](https://www.python.org/ftp/python/3.8.0/Python-3.8.0.tar.xz)
+
+To install DeepStack kindly check this [page](https://docs.deepstack.cc/object-detection/index.html) and [DeepStack Python SDK](https://docs.deepstack.cc/python-sdk/index.html)  which allows you to use DeepStack APIs.
+
+In order to install all the application's dependencies, run this command: 
+```
+pip install -r requiremnets.txt
+```
+
+## Run Web Application
+To run this streamlit application **locally** , execute the following command in the app folder
+
+First run Deepstack with all the apis if you prefer
+```
+deepstack --VISION-DETECTION True --VISION-FACE True --MODELSTORE-DETECTION "path to the custom action model" --PORT 80
```

Comment:
Replace this command with the `docker command` for running DeepStack and link back to the DeepStack documentation for running with other methods and platforms.